mdk/src/main/resources/pack.mcmeta

```diff
@@ -0,0 +1,11 @@
+# http://minecraft.gamepedia.com/Resource_pack#Contents
+# This will not show up in GuiResourcePackList
+{
+    "pack": {
+        # Unused, but required.
+        "description": "examplemod resources",
+        # A pack_format of "3" should be used starting with Minecraft 1.11. All resources, including language files, should be lowercase (eg: en_us.lang).
+        # A pack_format of "2" will load your mod resources with LegacyV2Adapter, which requires language files to have uppercase letters (eg: en_US.lang).
```

Comment:
In Minecraft itself, Just GSON, but not all GSON support comments it has to be opted in.
But this file is read by other things like launchers and stuff.
When I said "comment" I meant literally a field named "comment" as thats valid syntax it just means nothing.